src/bin/rustfmt.rs

```diff
@@ -138,6 +152,32 @@ fn determine_operation<I>(opts: &Options, args: I) -> Operation
         return Operation::Help;
     }
 
+    // if no file argument is supplied, read from stdin
+    if matches.free.len() != 1 {
```

Comment:
I'd prefer `== 0` here - I can imagine we might want to make > 1 args work some day, or give an error message rather than reading from stdin